stock_inventory_verification_request/models/stock_slot_verification_request.py

```diff
@@ -120,6 +124,27 @@ def _compute_created_inventory_count(self):
     )
     created_inventory_count = fields.Integer(compute="_compute_created_inventory_count")
 
+    @api.depends("location_id", "product_id", "lot_id", "state")
+    def _compute_involved_move_lines(self):
+        for rec in self:
+            # Solo calcular si el estado es 'open'
```

Comment:
Comments better in English

Reply:
sorry I missed this personal comment. Petition attended.

Thank you @AaronHForgeFlow

Reply:
I would go one step further, why do you need a comment that states what the code does just right into the next line? I'm not a big friend of comments that explain what code does, an useful comment would be one that explains **why** something is done. So in this case for me the comment can be removed.

Reply:
Comment attended

Thank you @LoisRForgeFlow